.github/workflows/reusable-web-e2e.yml

```diff
@@ -123,26 +123,6 @@ jobs:
           install: false
           config-file: cypress.config.ts
 
-      - name: Cypress run component tests
-        uses: cypress-io/github-action@v4
-        env:
-          NODE_ENV: test
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          CYPRESS_RECORD_KEY: ${{ secrets.cypress_key }}
-        with:
-          working-directory: apps/web
-          browser: chrome
-          component: true
-          record: true
-          install: false
-          config-file: cypress.config.ts
-
-      - uses: actions/upload-artifact@v1
-        if: failure()
-        with:
-          name: cypress-screenshots
-          path: apps/web/cypress/screenshots
```

Comment:
Should we return this to here? so the web e2e screenshots will be uploaded too

Reply:
oh, you are totally right I removed it by mistake while removing the component test.

Reply:
good catch!